great_expectations/datasource/data_connector/inferred_asset_dbfs_data_connector.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import Optional
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.datasource.data_connector import (
+    InferredAssetFilesystemDataConnector,
+)
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
+
+
+class InferredAssetDBFSDataConnector(InferredAssetFilesystemDataConnector):
+    """
+    Extension of InferredAssetFilesystemDataConnector used to connect to data on a DBFS filesystem.
+
+    The InferredAssetDBFSDataConnector is one of two classes (ConfiguredAssetDBFSDataConnector being the
+    other one) designed for connecting to data on a DBFS filesystem. It connects to assets
+    inferred from directory and file name by default_regex and glob_directive.
+
+    InferredAssetDBFSDataConnector that operates on file paths and determines
+    the data_asset_name implicitly (e.g., through the combination of the regular expressions pattern and group names)
+    """
+
+    def __init__(
+        self,
+        name: str,
+        datasource_name: str,
+        base_directory: str,
+        execution_engine: Optional[ExecutionEngine] = None,
```

Comment:
execution_engine would then be required here, too

Reply:
Resolved in https://github.com/great-expectations/great_expectations/pull/3659/commits/4c0f3b5623d6e94673fd4235823db879e154263f